components/ArtPiecePreview/index.js

```diff
@@ -2,7 +2,6 @@ import Image from "next/image";
 import styled from "styled-components";
 
 export default function ArtPiecePreview({ image, title, artist }) {
-  console.log(image);
   return (
     <StyledListItem>
       <Image src={image} alt="Art Picture" width={300} height={300} />
```

Comment:
You could use a piece of information from the array for the value of the alt attribute as well. "Art Picture" is very generic.